file_stream/file_stream_test.go

```diff
@@ -0,0 +1,101 @@
+package file_stream
+
+import (
+	"context"
+	"io"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestFileStream_MoreBytes(t *testing.T) {
+	ctx := context.Background()
+
+	Convey("file_stream.MoreBytes()", t, func() {
+		file_name := "/tmp/tail_test.log"
+		f, _ := os.Create(file_name)
+		init_content := "hello, world!\n"
+		f.WriteString(init_content)
+
+		stream := NewFileStream(file_name, ctx)
+
+		Convey("Should return a full buffer when possible", func() {
+			f.WriteString("xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx")
```

Comment:
`strings.Repeat("x", 100)` would make more sense here.

I _assume_ there's 100 `x`s there, but I'm not very interested in counting them if this test starts to fail 😅

Reply:
TIL about strings.Repeat